src/pages/index.js

```diff
@@ -0,0 +1,393 @@
+headerImageSrc.src = headerSrc;
+// const avatarImageSrc = document.getElementById("profile-avatar");
+// avatarImageSrc.src = profileSrc;
+const editProfileIconSrc = document.getElementById("edit-profile-icon");
+editProfileIconSrc.src = editIconSrc;
+const addCardIcon = document.getElementById("add-card-icon");
+addCardIcon.src = plusIconSrc;
+const closeModalIconSrc = document.getElementById("edit-modal-close-icon");
+closeModalIconSrc.src = closeIconSrc;
+const addCardModalCloseIconSrc = document.getElementById("add-card-modal-close-icon");
+addCardModalCloseIconSrc.src = closeIconSrc;
+const editProfileAvatarIcon = document.getElementById("profile-avatar-edit-icon");
+editProfileAvatarIcon.src = editProfileAvatarIconSrc;
+const deleteCardIcon = document.getElementById("delete-modal-close-btn");
+deleteCardIcon.src = deleteCardIconSrc;
+profileEditbutton.addEventListener('click', () => {
+  editModalNameInput.value = profileName.textContent;
+  editModalDescriptionInput.value = profileDescription.textContent;
+  resetValidation(editFormElement, editModalNameInput, editModalDescriptionInput);
+  openModal(editModal);
+});
+
+editModalCloseBtn.addEventListener('click', () => {
+  closeModal(editModal);
+});
+
+profileAddbutton.addEventListener('click', () => {
+  openModal(cardModal);
+});
+cardModalCloseBtn.addEventListener('click', () => {
+  closeModal(cardModal);
+});
+
+previewModalCloseBtn.addEventListener('click', () => {
+  closeModal(previewModal);
+});
+
+// add event listener to the avatar image to open the modal
+avatarModalBtn.addEventListener('click', () => {
+  openModal(avatarModal);
+});
+avatarModalCloseBtn.addEventListener('click', () => {
+  closeModal(avatarModal);
+}
+);
+
+deleteModalCloseBtn.addEventListener('click', () => {
+  closeModal(deleteModal);
+}
+);
+
+deleteModalCancelBtn.addEventListener('click', () => {
+  closeModal(deleteModal);
+}
+);
+
+avatarForm.addEventListener("submit", handleAvatarSubmit);
+
+deleteForm.addEventListener('submit', handleDeleteSubmit);
+
+
+editFormElement.addEventListener('submit', handleEditFormSubmit);
+cardForm.addEventListener('submit', handleAddCardFormSubmit);
+/*for (let i = 0; i < initialCards.length; i++) {
+  const cardElement = getCardElement(initialCards[i]);
+  cardsList.prepend(cardElement);
+} */
+
+// initialCards.forEach((item, i, arr) => {
+//   const cardElement = getCardElement(item);
+//   cardsList.prepend(cardElement);
+// });
+
+function handleOverlayClick(evt) {
+  if (evt.target.classList.contains('modal_opened')) {
+    closeModal(evt.target);
+  }
+}
+
+document.querySelectorAll('.modal').forEach((modal) => {
+  modal.addEventListener('mousedown', handleOverlayClick);
+});
+
+function handleEscapeKey(evt) {
+  if (evt.key === 'Escape') {
+    const openedModal = document.querySelector('.modal_opened');
+    if (openedModal) {
+      closeModal(openedModal);
+    }
+  }
+}
+
+
+document.addEventListener('keydown', handleEscapeKey);
```

Comment:
`document.addEventListener('keydown', handleEscapeKey);` 


Please, delete it from lne 390 because it's already added inside `openModal`